BenchmarkSamples/benchmark/proguard-rules.pro

```diff
@@ -19,3 +19,19 @@
 # If you keep the line number information, uncomment this to
```

Comment:
call this benchmark-proguard-rules.pro

We can also think about pulling this into the library later on.

Reply:
:+1: